test/arbitrage.test.js

```diff
@@ -182,32 +266,150 @@ test('all no true', t => {
   t.true(r.all(x) == false)
 })
 
+test('is_equal 1', t => {
+  var act = r.is_equal(1,1)
+  var exp = [ true ]
+  t.true(r.all(exp) == true)
```

Comment:
This has a redundant expression. Remove for clarity. If you don't know which part is redundant, shame on you.